timewarp_gl/plugin.cpp

```diff
@@ -539,9 +541,24 @@ class timewarp_gl : public threadloop {
 		// Now that we have the most recent swap time, we can publish the new estimate.
 		_m_vsync_estimate->put(new time_type(GetNextSwapTimeEstimate()));
 
+		std::chrono::nanoseconds imu_to_display = lastSwapTime - latest_pose.pose.sensor_time;
+		std::chrono::nanoseconds predict_to_display = lastSwapTime - latest_pose.predict_computed_time;
+		std::chrono::nanoseconds render_to_display = lastSwapTime - most_recent_frame->render_time;
+
+		mtp_logger.log(record{mtp_record, {
+			{iteration_no},
+			{lastSwapTime},
+			{imu_to_display},
+			{predict_to_display},
+			{render_to_display},
+		}});
+
 #ifndef NDEBUG
 		auto afterSwap = glfwGetTime();
 		printf("\033[1;36m[TIMEWARP]\033[0m Swap time: %5fms\n", (float)(afterSwap - beforeSwap) * 1000);
+		printf("\033[1;36m[TIMEWARP]\033[0m Motion-to-display latency: %3f ms\n", float(mtp.count()) / 1e6);
```

Comment:
There is a compilation error here. `mpt.count()` should be `imu_to_display.count()`.

Also, we print both MTP and PTD. Might as well print RTD too.